### PR TITLE
cleaning up models and names

### PR DESCRIPTION
**Pull request by chenkc805**
## Test plan
test locally

### CODE REVIEW

**Comment by chenkc805** [comment]
I couldn't get this to build locally because I was getting this error, which doesn't seem relevant to the code I was changing at all:

```
[1] error during build:
[1] Error: [vite]: Rollup failed to resolve import "observable-fns" from "/Users/kevinchen/Documents/cody/lib/shared/src/experimentation/FeatureFlagProvider.ts".
```

File: vscode/test/e2e/chat-input.test.ts [hunk 1]
```diff
@@ -159,10 +159,10 @@ test.extend<DotcomUrlOverride>({ dotcomUrl: mockServer.SERVER_URL })(
         await expect(humanRow0.toolbar.mention).toBeVisible()
         await expect(humanRow0.toolbar.modelSelector).toBeVisible()
         await expect(humanRow0.toolbar.submit).toBeVisible()
-        await expect(chatPanel.getByText('Optimized for Accuracy')).not.toBeVisible()
+        await expect(chatPanel.getByText('More powerful models')).not.toBeVisible()
```
Comment (sqs) [inline vscode/test/e2e/chat-input.test.ts:162]: UI text here is capitalized per the style of most editors

**Review by sqs — APPROVED** [review]
awesome!

To fix that error locally, run `pnpm install` in the repo.

File: lib/shared/src/models/dotcom.ts [hunk 2]
```diff
@@ -43,23 +43,15 @@ export const DEFAULT_DOT_COM_MODELS = [
...
-        tags: [ModelTag.Gateway, ModelTag.Accuracy, ModelTag.Recommended, ModelTag.Free],
-    },
-    {
-        title: 'Claude 3 Sonnet',
-        model: 'anthropic/claude-3-sonnet-20240229',
-        provider: 'Anthropic',
-        usage: [ModelUsage.Chat, ModelUsage.Edit],
-        contextWindow: expandedContextWindow,
-        tags: [ModelTag.Gateway, ModelTag.Balanced],
+        tags: [ModelTag.Gateway, ModelTag.Balanced, ModelTag.Recommended, ModelTag.Free],
```
Comment (abeatrix) [inline lib/shared/src/models/dotcom.ts:46]: Does 3.5 sonnet belongs to Power or Balance group?

**Review by abeatrix — APPROVED** [review]
Apart from the comment from sqs on the UI change, the rest looks good to me. 
Just need to update the kotlin bindings for the tests to pass 👍 approve to unblock.

File: lib/shared/src/models/dotcom.ts (hunk 2 above)
Comment (chenkc805) [reply lib/shared/src/models/dotcom.ts:46]: Balance. Technically it _is_ more powerful than Opus but it's marketed as the "best of both worlds" model between haiku (fast but lighter) and opus (slow but powerful)

**Pull request merged**